R/writeDBResultsTo.R

```diff
@@ -36,43 +36,27 @@ writeDBResultsToJson <- function(connection,
                                     outputFile) {
     startTime <- Sys.time()
```

Comment:
can remove start and end time, and the delta variable now we're not it :)

Reply:
Done!